Source/SwiftLintFramework/Rules/CyclomaticComplexityRule.swift

```diff
@@ -36,8 +40,7 @@ public struct CyclomaticComplexityRule: ASTRule, ConfigProviderRule {
             return []
         }
 
-        let substructure = dictionary["key.substructure"] as? [SourceKitRepresentable] ?? []
-        let complexity = measureComplexity(substructure) + 1
+        let complexity = measureComplexity(file, dictionary: dictionary) + 1
```

Comment:
I think this `+ 1` is wrong. See my comment here: https://github.com/realm/SwiftLint/issues/461#issuecomment-178115875